packages/kbn-esql-validation-autocomplete/src/autocomplete/factories.ts

```diff
@@ -272,7 +273,7 @@ export const buildPoliciesDefinitions = (
 ): SuggestionRawDefinition[] =>
   policies.map(({ name: label, sourceIndices }) => ({
     label,
-    text: getSafeInsertText(label, { dashSupported: true }),
+    text: getSafeInsertText(label, { dashSupported: true }) + ' ',
```

Comment:
I'm starting to think we should make this `' '` a constant for clarity and easy searching. So something like `OPEN_SUGGESTION = ' '` and then `text + OPEN_SUGGESTION`

Reply:
The `' '` is not what opens the suggestion menu. That is the `command` property.

Reply:
Sorry, I meant advance cursor. Switched it around in my brain. And this is just a thought, not a blocker.